bin/ch/Helpers.cpp

```diff
@@ -133,9 +58,9 @@ HRESULT Helpers::LoadScriptFromFile(LPCSTR filename, LPCWSTR& contents, bool* is
     //
     // Read the entire content as a binary block.
     //
-    fread((void*)contentsRaw, sizeof(char), lengthBytes, file);
+    fread(pRawBytes, sizeof(BYTE), lengthBytes, file);
     fclose(file);
-    *(WCHAR*)((byte*)contentsRaw + lengthBytes) = _u('\0'); // Null terminate it. Could be LPCWSTR.
+    *reinterpret_cast<WCHAR*>(pRawBytes + lengthBytes) = 0; // Null terminate it. Could be UTF16
```

Comment:
I haven't tried yet but most probably compiler already knows `pRawBytes` is  `sizeof(BYTE)`!! x (lengthBytes) away from it's starting point. So it does the rest hence it wouldn't give trouble during the runtime. However comparing to old code, it is miss-leading. Adding `(BYTE*)` right before `pRawBytes` would make it more clear.

Old code has C style reinterpret_cast and yours C++. IMO, in both cases, I'm having trouble when I see char memory is being reinterpret cast to char16 (Endiannes ?) even though there are couple of general `harmless` usages like this. 

So it is `okay` to put null termination in a single line but IMO making it clear would give less trouble to reader.

> however we only support up to WCHAR - 2 bytes
I guess you mean (at least) 2 bytes ? Since, there is no guarantee that it will be 2 bytes.